server/routerlicious/packages/routerlicious-base/src/alfred/app.ts

```diff
@@ -137,6 +137,15 @@ export function create(
 					if (req.body?.isEphemeralContainer !== undefined) {
 						additionalProperties.isEphemeralContainer = req.body.isEphemeralContainer;
 					}
+					const customHeadersToLog = (config.get("customHeadersToLog") as string[]) ?? [];
```

Comment:
Some headers might contain PII and we should be careful what header names are being added in this config. When enabling this in config files it might be worth to add a comment about that.